src/autocomplete.js

```diff
@@ -1,86 +1,116 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
-import mitt from 'mitt'
 
 import Button from './button'
 import Controller from './controller'
 import Input from './input'
 import Item from './item'
-import Menu from './menu'
-import {AUTOCOMPLETE_CONTEXT} from './constants'
-import {cbToCb, compose} from './utils'
+import Status from './status'
+import {cbToCb, compose, scrollIntoView} from './utils'
+import AUTOCOMPLETE_CONTEXT from './context'
 
 class Autocomplete extends Component {
   static Button = Button
   static Controller = Controller
   static Input = Input
   static Item = Item
-  static Menu = Menu
   static childContextTypes = {
     [AUTOCOMPLETE_CONTEXT]: PropTypes.object.isRequired,
   }
 
   static propTypes = {
-    ref: PropTypes.func,
+    children: PropTypes.node.isRequired,
+    defaultHighlightedIndex: PropTypes.number,
+    getA11yStatusMessage: PropTypes.func,
     onChange: PropTypes.func.isRequired,
+    ref: PropTypes.func,
```

Comment:
Actually, we should remove this proptype because we can't consume the `ref` prop anyway.